src/api/spec/jobs/measurements_job_spec.rb

```diff
@@ -0,0 +1,16 @@
+RSpec.describe MeasurementsJob do
+  describe 'token workflows' do
+    before do
+      allow(RabbitmqBus).to receive(:send_to_bus)
+      allow(CONFIG).to receive(:[]).and_return({ 'amqp_options' => true })
```

Comment:
```suggestion
      stub_const('CONFIG', CONFIG.merge('amqp_options' => { dummy: 1 }))
```

I prefer to treat CONFIG as it is, a constant. We already do something similar here: https://github.com/openSUSE/open-build-service/blob/4e2788e36a1fd19954cda239267510ca438591cb/src/api/spec/support/rabbitmq.rb#L28